at_commons/lib/src/keystore/at_key_builder_impl.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:at_commons/at_commons.dart';
+import 'package:at_commons/src/keystore/at_key_builder.dart';
+
+/// Abstract base class for the KeyBuilder
+abstract class AbstractKeyBuilder implements KeyBuilder {
```

Comment:
there are some default values in metadata that has to be set at key creation e.g createdAt,  createdBy..should they be set in KeyBuilder?

Reply:
At the moment, the default values are being set in at metadata builder in at_persistence_secondary_server. Will discuss with Jagan on moving the defaults to KeyBuilder